Platform/CoffeelakeBoardPkg/Script/StitchIfwi.py

```diff
@@ -368,6 +368,52 @@ def sign_binary(infile, stitch_dir, cfg_var):
     hash = bytearray(open('output/input/pubkey'+'.hash','rb').read())
     print(''.join('%02X ' % b for b in hash))
 
+def update_tpm_type(tpm, tree):
+
+  # Enable ptt/dtpm
+  if tpm == "dtpm":
+      #PttConfiguration
+      node = tree.find('./PlatformProtection/IntelPttConfiguration/PttSupported')
```

Comment:
Also, file path can be generalized with os.path.join().

Reply:
This is an xml entry. I would update with dict methods.